server/src/main/java/org/apache/iotdb/db/qp/physical/sys/AuthorPlan.java

```diff
@@ -254,6 +254,15 @@ public String toString() {
     return ret;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    List<String> ret = new ArrayList<>();
+    if (nodeName != null) {
+      ret.add(nodeName.getFullPath());
+    }
+    return ret;
```

Comment:
`return nodeName != null ? Collections.singletonList(nodeName.getFullPath()) : Collections.emptyList()`.

Reply:
I'll change it. Thanks for your suggestion.